script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -567,12 +590,13 @@ Arcueid plunges her bare hand into the ground and lifts a chunk of earth,
 [sha:1c1bdb5141eefd033c545938ebc034ad05e4ed4f]{
 -- Page 636, Offset 10367.
 -- そのまま、痛いじゃない、と言わんばかりに岩盤をヴローヴめがけて投げつける。
-She then aims the bedrock at Vlov and throws it as if to say 'that didn't hurt at all'.
+She then hurls the whole chunk of bedrock at Vlov,
+ almost like she's complaining about how much it hurt.
 }
 [sha:ec8e844cbcb560f41707db1a549b6878e278266a]{
 -- Page 637, Offset 10368.
 -- アルクェイドの<岩盤|ちからわざ>を、巨大な槍が打ち砕く。
-The huge lance crushes the bedrock thrown by Arcueid.
+The bedrock Arcueid threw at him is shattered by his huge lance.
```

Comment:
threw > throws

Reply:
She's already thrown it at this point, pretty sure

Reply:
somehow, it feels right with throws
I think it's because it hasn't landed